cmd/rekor-server/e2e_test.go

```diff
@@ -221,3 +221,158 @@ func getTreeID(t *testing.T) int64 {
 func TestSearchNoEntriesRC1(t *testing.T) {
 	util.RunCliErr(t, "search", "--email", "noone@internetz.com")
 }
+func TestHostnameInSTH(t *testing.T) {
+	// get ID of container
+	rekorContainerID := strings.Trim(util.Run(t, "", "docker", "ps", "-q", "-f", "name=rekor-server"), "\n")
+	resp, err := http.Get(fmt.Sprintf("%s/api/v1/log", rekorServer()))
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !strings.Contains(string(body), fmt.Sprintf(" %s ", rekorContainerID)) {
+		t.Errorf("logInfo does not contain the hostname (%v) of the rekor-server container: %v", rekorContainerID, string(body))
+	}
+	if strings.Contains(string(body), "rekor.sigstore.dev") {
+		t.Errorf("logInfo contains rekor.sigstore.dev which should not be set by default")
+	}
+}
+func rekorServer() string {
+	if s := os.Getenv("REKOR_SERVER"); s != "" {
+		return s
+	}
+	return "http://localhost:3000"
+}
+func TestSearchSHA512(t *testing.T) {
+	sha512 := "c7694a1112ea1404a3c5852bdda04c2cc224b3567ef6ceb8204dbf2b382daacfc6837ee2ed9d5b82c90b880a3c7289778dbd5a8c2c08193459bcf7bd44581ed0"
+	var out string
+	out = util.RunCli(t, "upload", "--type", "intoto:0.0.2",
+		"--artifact", "tests/envelope.sha512",
+		"--pki-format", "x509",
+		"--public-key", "tests/test_sha512.pub")
+	util.OutputContains(t, out, "Created entry at")
+	uuid := getUUIDFromTimestampOutput(t, out)
+	out = util.RunCli(t, "search", "--sha", fmt.Sprintf("sha512:%s", sha512))
+	util.OutputContains(t, out, uuid)
+}
+func getUUIDFromTimestampOutput(t *testing.T, out string) string {
+	t.Helper()
+	// Output looks like "Created entry at index X, available at $URL/UUID", so grab the UUID:
+	urlTokens := strings.Split(strings.TrimSpace(out), "\n")
+	return getUUIDFromUploadOutput(t, urlTokens[len(urlTokens)-1])
+}
+func getUUIDFromUploadOutput(t *testing.T, out string) string {
+	t.Helper()
+	// Output looks like "Artifact timestamped at ...\m Wrote response \n Created entry at index X, available at $URL/UUID", so grab the UUID:
+	urlTokens := strings.Split(strings.TrimSpace(out), " ")
+	url := urlTokens[len(urlTokens)-1]
+	splitUrl := strings.Split(url, "/")
+	return splitUrl[len(splitUrl)-1]
+}
+func TestLogInfo(t *testing.T) {
+	// TODO: figure out some way to check the length, add something, and make sure the length increments!
+	out := util.RunCli(t, "loginfo")
+	util.OutputContains(t, out, "Verification Successful!")
+}
```

Comment:
What is the thought process for having 1 `func` in a file?

Reply:
More for consistency sake - if we have other CLI commands' tests in individual files, I don't see why we wouldn't do it here (although since there are no `loginfo` specific arguments I can't see a whole lot more being added).

Reply:
Agreed! 👍